Cairo Desktop/Cairo Desktop/CairoSettingsWindow.xaml

```diff
@@ -62,311 +73,606 @@
                         </StackPanel>
                     </StackPanel>
-                    <StackPanel Orientation="Horizontal" Style="{StaticResource RadioGroup}">
-                        <TextBlock Text="{Binding Path=(l10n:DisplayString.sSettings_Desktop_LabelPosition)}" VerticalAlignment="Top" Margin="5,3,5,4" />
+                    <StackPanel Orientation="Horizontal"
+                                Style="{StaticResource RadioGroup}">
+                        <TextBlock Text="{Binding Path=(l10n:DisplayString.sSettings_Desktop_LabelPosition)}"
+                                   VerticalAlignment="Top"
+                                   Margin="5,3,5,4" />
                         <StackPanel Orientation="Vertical">
-                            <RadioButton GroupName="rdoDesktopLabelPos" Name="radDesktopLabelPos0" Click="radDesktopLabelPos0_Click">
+                            <RadioButton GroupName="rdoDesktopLabelPos"
+                                         Name="radDesktopLabelPos0"
+                                         Click="radDesktopLabelPos0_Click">
                                 <Label Content="{Binding Path=(l10n:DisplayString.sSettings_Desktop_LabelPositionRight)}" />
                             </RadioButton>
-                            <RadioButton GroupName="rdoDesktopLabelPos" Name="radDesktopLabelPos1" Click="radDesktopLabelPos1_Click">
+                            <RadioButton GroupName="rdoDesktopLabelPos"
+                                         Name="radDesktopLabelPos1"
+                                         Click="radDesktopLabelPos1_Click">
                                 <Label Content="{Binding Path=(l10n:DisplayString.sSettings_Desktop_LabelPositionBottom)}" />
                             </RadioButton>
                         </StackPanel>
                     </StackPanel>
-                    <TextBlock Text="{Binding Path=(l10n:DisplayString.sSettings_Desktop_DynamicDesktop)}" Style="{StaticResource SettingGroupHeader}" />
-                    <CheckBox IsChecked="{Binding Path=EnableDynamicDesktop}"
-                        Click="CheckBox_Click" Name="chkEnableDynamicDesktop">
+                    <TextBlock Text="{Binding Path=(l10n:DisplayString.sSettings_Desktop_DynamicDesktop)}"
+                               Style="{StaticResource SettingGroupHeader}" />
+                    <CheckBox IsChecked="{Binding Source={x:Static Settings:Settings.Instance}, Path=EnableDynamicDesktop, UpdateSourceTrigger=PropertyChanged}"
+                              Click="CheckBox_Click"
+                              Name="chkEnableDynamicDesktop">
                         <Label Content="{Binding Path=(l10n:DisplayString.sSettings_Desktop_EnableDynamicDesktop)}" />
                     </CheckBox>
+
+                    <StackPanel Name="cairoBackgroundSettingsStackPanel"
+                                Orientation="Vertical">
+                        <TextBlock Text="Desktop Background Settings"
```

Comment:
Could the new strings here be made localizable?

Reply:
This is added.